mindsdb/integrations/handlers/youtube_handler/youtube_tables.py

```diff
@@ -148,3 +145,59 @@ def call_youtube_comments_api(self,video_id):
 
 
         return all_youtube_comments_df
+
+
+class YoutubeChannelTable(APITable):
+
+    """Youtube Channel Info  by channel id Table implementation"""
+
+    def select(self, query: ast.Select) -> pd.DataFrame:
+        select_statement_parser = SELECTQueryParser(
+            query,
+            'channel',
+            self.get_columns()
+        )
+
+        selected_columns, where_conditions, order_by_conditions, result_limit = select_statement_parser.parse_query()
+
+        channel_id = None
+        for op, arg1, arg2 in where_conditions:
+            if arg1 == 'channel_id':
+                if op == '=':
+                    channel_id = arg2
+                    break
+                else:
+                    raise NotImplementedError("Only '=' operator is supported for channel_id column.")
+
+        if not channel_id:
```

Comment:
Same question as before with your other PR, is it always mandatory to include the channel ID?

Reply:
If it absolutely is, please make sure you mention this in the README.

Reply:
Channel ID is needed. Updated readme.